### PR TITLE
11.0 mig l10n siret

### PR DESCRIPTION
**Pull request by fmdl**

### CODE REVIEW

File: l10n_fr_siret/models/__init__.py [hunk 1]
```diff
@@ -0,0 +1,23 @@
+# -*- coding: utf-8 -*-
+##############################################################################
```
Comment (elicoidal) [inline l10n_fr_siret/models/__init__.py]: Please remove headers in `__init__.py` files

File: l10n_fr_siret/models/partner.py [hunk 2]
```diff
@@ -0,0 +1,111 @@
+# -*- coding: utf-8 -*-
```
Comment (elicoidal) [inline l10n_fr_siret/models/partner.py]: remove coding line and switch to short headers

File: l10n_fr_siret/models/partner.py [hunk 3]
```diff
@@ -0,0 +1,111 @@
...
+
+    @api.constrains('siren', 'nic')
+    def _check_siret(self):
+        """Check the SIREN's and NIC's keys (last digits)"""
+        for rec in self:
+            if rec.nic:
+                # Check the NIC type and length
+                if not rec.nic.isdecimal() or len(rec.nic) != 5:
+                    raise UserError(
+                        _("The NIC '%s' is incorrect: it must be have "
```
Comment (elicoidal) [inline l10n_fr_siret/models/partner.py]: "it must have exactly 5 digits."

File: l10n_fr_siret/models/partner.py [hunk 4]
```diff
@@ -0,0 +1,111 @@
...
+
+    def _commercial_fields(self):
+        res = super(Partner, self)._commercial_fields()
+        res += ['siren', 'nic']
+        return res
+
+    siren = fields.Char(
+        string='SIREN', size=9, track_visibility='onchange',
+        help="The SIREN number is the official identity "
+        "number of the company in France. It makes "
```
Comment (elicoidal) [inline l10n_fr_siret/models/partner.py]: s/makes/composes

File: l10n_fr_siret/models/partner.py [hunk 5]
```diff
@@ -0,0 +1,111 @@
...
+        "the first 9 digits of the SIRET number.")
+    nic = fields.Char(
+        string='NIC', size=5, track_visibility='onchange',
+        help="The NIC number is the official rank number "
+        "of this office in the company in France. It "
+        "makes the last 5 digits of the SIRET "
```
Comment (elicoidal) [inline l10n_fr_siret/models/partner.py]: s/makes/composes

File: l10n_fr_siret/README.rst [hunk 6]
```diff
@@ -0,0 +1,60 @@
...
+
+On the Partner form, users will be able to enter the SIREN
+and NIC numbers, and the SIRET number will be calculated automatically.
+
+The last digits of the SIREN and NIC are control keys:
+Odoo will check their validity when partners are recorded.
+
+.. image:: https://odoo-community.org/website/image/ir.attachment/5784_f2813bd/datas
+   :alt: Try me on Runbot
+   :target: https://runbot.odoo-community.org/runbot/121/10.0
```
Comment (elicoidal) [inline l10n_fr_siret/README.rst]: please update 10->11

**Review by elicoidal — CHANGES_REQUESTED** [review]
almost there

File: l10n_fr_siret/models/partner.py (hunk 5 above)
Comment (fmdl) [reply l10n_fr_siret/models/partner.py]: What do you mean by s/makes/composes ? Regards

Comment (elicoidal) [reply l10n_fr_siret/models/partner.py]: sorry: replace `makes` by `composes`

**Comment by njeudy** [comment]
@fmdl just this to review: l10n_fr_siret/models/partner.py:64:5: E303 too many blank lines (2)

**Comment by njeudy** [comment]
@elicoidal can you review ?

**Review by njeudy — APPROVED** [review]
Ok for me tested in live database

**Review by elicoidal — APPROVED** [review]
LGTM (no tests)

**Pull request merged**